src/app.js

```diff
@@ -15,6 +19,21 @@ const PORT = process.env.APPLICATION_PORT
 //middleware definition
 app.use(bodyParser.json({ limit: '50MB' }))
 app.use(bodyParser.urlencoded({ limit: '50MB', extended: false }))
+app.use(correlationIdMiddleware)
+
+if (process.env.ENABLE_LOG === 'true') {
```

Comment:
@nevil-mathew Can we use ENABLE LOG or DIsable Log any one, we can delete which is not required

Reply:
Removed the if statement , since logs will be disable with npm package